### ISSUE TITLE
console.log not reporting

### ISSUE THREAD

**Issue by gitdisrupt** [driftyco/cordova-plugin-wkwebview-engine#133]
On app launch, on device running iOS 10, i get the following error in the console:

`Failed to load resource: the server responded with a status of 404 (Not Found).`

All subsequent calls to console.log from ionic are not displaying in the console, although the app is running OK without issue otherwise.

My system information:
```
Cordova CLI: 6.5.0 
Ionic Framework Version: 3.1.1
Ionic CLI Version: 2.2.3
Ionic App Lib Version: 2.2.1
Ionic App Scripts Version: 1.3.4
ios-deploy version: 1.9.1 
ios-sim version: 5.0.13 
OS: macOS Sierra
Node Version: v7.9.0
Xcode version: Xcode 8.3 Build version 8W120l
```

**Comment by gitdisrupt** [comment]
STRANGE ... well i uninstalled and reinstalled and now the console is working ....

Although - i am still seeing the app attempt to load main.js.map from a path located in my working directory on my desktop.

`Failed to load resource: the server responded with a status of 404 (Not Found). http://localhost:8080/Users/mspro/Desktop/testapp/www/build/main.js.map`

**Comment by DavidStrausz** [comment]
If you look at the last line of www/build/main.js you will see that the sourceMappingUrl contains an absolute path (I opend a bug for that with ionic-app-scripts). A production build does (of course) not contain a sourcemap and that is why the webview can't load it.

**Comment by gitdisrupt** [comment]
yes understood on why the production build does not and should not include it
